packages/manager/apps/vrack-services/src/pages/dashboard/[id]/_layout.tsx

```diff
@@ -58,7 +58,7 @@ export default function DashboardPage() {
   ];
 
   if (results[0].status === 'success' && results[0]?.data?.status !== 200) {
-    return <ErrorBanner error={results[0].data} />;
+    return <ErrorPage error={results[0].data} />;
```

Comment:
you can pass error from results const { error } = results[0]

Reply:
I will fix when I rebase on dev-transformation the main branch